root/static/styles/layout.less

```diff
@@ -136,6 +136,11 @@ pre code {
 /* For general right aligning, and especially for barcodes */
 .align-right, .barcode-cell { text-align: right; }
 
+#expand-medium-helper {
+    color: @dark-grey;
+    opacity: 0.5;
```

Comment:
This looks almost invisible on my screen. 😄TBH I don't see a need to change the text color or opacity.